### ISSUE TITLE
No Interactive Shell using Gradle 6.3 and JDK 14

### ISSUE THREAD

**Issue by alex-aeiou** [mrsarm/jshell-plugin#1]
Added the plugin in the following version:

```gradle
id "com.github.mrsarm.jshell.plugin" version "1.0.0-RC1"
```

Running the command:

 ```bash
./gradlew --no-daemon build -x test --console plain jshell
```

Leads to it successfully building and starting JShell but i doesn't grant me interactive input and stops as "BUILD SUCCESSFUL" immediately after:

```
To honour the JVM settings for this build a new JVM will be forked. Please consider using the daemon: https://docs.gradle.org/6.3/userguide/gradle_daemon.html.
Daemon will be stopped at the end of the build stopping after processing

> Configure project :
:jshell task :classes not found, be sure to compile the project first

> Task :compileJava UP-TO-DATE
> Task :processResources UP-TO-DATE
> Task :classes UP-TO-DATE
> Task :bootJar UP-TO-DATE
> Task :jar SKIPPED
> Task :assemble UP-TO-DATE
> Task :check
> Task :build

> Task :jshell
|  Welcome to JShell -- Version 14
|  For an introduction type: /help intro

BUILD SUCCESSFUL in 11s
4 actionable tasks: 1 executed, 3 up-to-date
```

Some infos about my system:

```
➜  service java --version
openjdk 14 2020-03-17
OpenJDK Runtime Environment (build 14+36-1461)
OpenJDK 64-Bit Server VM (build 14+36-1461, mixed mode, sharing)
➜  service ✗ ./gradlew --version

------------------------------------------------------------
Gradle 6.3
------------------------------------------------------------

Build time:   2020-03-24 19:52:07 UTC
Revision:     bacd40b727b0130eeac8855ae3f9fd9a0b207c60

Kotlin:       1.3.70
Groovy:       2.5.10
Ant:          Apache Ant(TM) version 1.10.7 compiled on September 1 2019
JVM:          14 (Oracle Corporation 14+36-1461)
OS:           Mac OS X 10.15.3 x86_64
```

**Comment by mrsarm** [comment]
That is strange, anyway, I did not test the plugin yet with JDK 14, although I don't think is related with that, but I'm going to refactor part of the code this week to put the autocomplete of the jshell console to work again, and that refactor will make strong changes on how the console startups, so after that I'll let you know.

**Comment by mrsarm** [comment]
@alex-aeiou I have released the 1.0.0 version with some refactors that may solve your issue, could you test it again?

**Comment by arcesino** [comment]
This is also happening to me using Gradle 7.1 & JDK 11 (both oracle-jdk & open-jdk)

**Comment by mrsarm** [comment]
Patch released today, version [1.1.0](https://github.com/mrsarm/jshell-plugin/releases/tag/1.1.0).
